src/main/java/xpathes/Xpathes.java

```diff
@@ -0,0 +1,110 @@
+    private WebElement enterToAccountLink;
+
+    @FindBy(xpath = "//button[@class='compare-button' and @type='button']")
+    private List<WebElement> comparisonButton;
+
+    @FindBy(xpath = "//button[@class='wish-button js-wish-button' and @type='button']")
+    private WebElement addToWishListButton;
+
+    @FindBy(xpath = "//ancestor::rz-city/button[@class='button--link header-cities__label']")
+    private WebElement chooseCity;
+
+    @FindBy(xpath = "//ancestor::button[contains(text(),'Сообщить, когда появится')]")
+    private WebElement outOfStackButton;
+
+    @FindBy(xpath = "(//span[@class='goods-tile__title'])/parent::a")
+    private List<WebElement> allProducts;
+
+    @FindBy(xpath = "//p[@class='main-auth__user-name']/parent::li")
+    private WebElement nameOfAccount;
+
+    @FindBy(xpath = "//div[@class='layout']/preceding::h1[@class='product__title']")
+    private WebElement productTitle;
+
+    @FindBy(xpath = "//p[@class='personal-data__value']/preceding::p[contains(text(),' petrovartem01061996@gmail.com ')]")
+    private WebElement personalEmailOfAccount;
+
+    @FindBy(xpath = "//h4[contains(text(),'Корзина пуста')]/preceding-sibling::img ")
+    private WebElement emptyCartImage;
+
+    @FindBy(xpath = "//h3[contains(text(),' У вас еще нет списков желаний ')]/preceding-sibling::img")
+    private WebElement wishListCapture;
+
+    @FindBy(xpath = "//div[contains(@class, 'main-stores')]//descendant::a")
+    private List<WebElement> appStoreAndGooglePlayApplications;
+
+    @FindBy(xpath = "//li[@class='main-auth__links-item']/descendant::a[contains(text(),'Просмотренные товары')]")
+    private WebElement viewProducts;
+
+    @FindBy(xpath = "//li[@class='main-auth__links-item']//following-sibling::a[contains(text(), 'Списки желаний')]")
+    private WebElement listOfWishes;
+
+    @FindBy(xpath = "//div[@class='menu-outer js-rz-fat-menu']//following-sibling::span[@class='menu-toggler__text']")
+    private WebElement mainProductsCatalogue;
+
+    @FindBy(xpath = "//a[@class='cart-product__title'][count(a)=0]")
+    private WebElement productTitleInCart;
+
+    @FindBy(xpath = "//button[@id='cartProductActions0'][count(button)=0]")
+    private WebElement additionalMenuInProductCart;
+
+    @FindBy(xpath = "//button[@class='button button--medium button--with-icon button--link context-menu-actions__button'] | //button[contains(text(),'Удалить')] ")
+    private WebElement deleteFromProductCartButton;
+
+    @FindBy(xpath = "//a[@class='popular-category__title'] | //span[contains(text(),' PlayStation 5 ')]")
+    private WebElement playStation5Category;
+
+    @FindBy(xpath = "//li[string-length(text()) >= 110]")
+    private List<WebElement> dignityOfNotebooks;
+
+    @FindBy(xpath = "//img[@height>=40]")
+    private List<WebElement> downloadFromGooglePlayOrAppStorePicture;
+
+    @FindBy(xpath = " //*[@class='goods-tile__price-value'][number()<=100]")
+    private WebElement priceLowerThan100;
```

Comment:
pls add comment on what page it can be checked

Reply:
``

> pls add comment on what page it can be checked
e.x https://rozetka.com.ua/search/?text=%D0%BD%D0%BE%D0%B6%D0%B8+%D0%BA%D1%83%D1%85%D0%BE%D0%BD%D0%BD%D1%8B%D0%B5+%D0%BD%D0%B0%D0%B1%D0%BE%D1%80%D1%8B